tests/integration/test_benchmark.py

```diff
@@ -211,7 +211,7 @@ def sample_from_synthesizer(synthesizer, n_samples):
     pd.testing.assert_series_equal(results['Synthesizer'], expected_synthesizers)
 
     assert set(results['Dataset']) == {'fake_companies'}
-    assert set(results['Dataset_Size_MB']) == {0.00128}
+    assert set(round(results['Dataset_Size_MB'], 5)) == {.00128}
```

Comment:
minor: alternatively you could use 
```python
assert np.isclose(a, b)
``` 
I think that seems more clear than rounding